packages/block-editor/src/components/iframe/index.js

```diff
@@ -104,25 +104,21 @@ function Iframe( {
 	contentRef,
 	children,
 	tabIndex = 0,
-	shouldZoom = false,
+	scale = 1,
+	frameSize = 0,
```

Comment:
These are the main things that needed to be restored since this component is exported from the package.